storage/storage_transport.go

```diff
@@ -284,8 +285,12 @@ func (s storageTransport) ValidatePolicyConfigurationScope(scope string) error {
 func verboseName(name reference.Named) string {
 	name = reference.TagNameOnly(name)
 	tag := ""
+	id := ""
 	if tagged, ok := name.(reference.NamedTagged); ok {
-		tag = tagged.Tag()
+		tag = ":" + tagged.Tag()
+	}
+	if canonical, ok := name.(reference.Canonical); ok {
+		id = "@" + canonical.Digest().Hex()
```

Comment:
This one is also reachable through `storageTransport.GetStoreImage`, which is, possibly, callable with any kind of `types.ImageReference`.  So, if `contaiers/storage` wants a hex string, this addition seems correct.